### PR TITLE
F186 orwu1 newloc

### PR DESCRIPTION
**Pull request by sidjtd**
This RPC deserves major props to Adam for pioneering through some tricky stuff in f178clinLoc. Found odd irregularity when passing in ['', -1] as arguments so addressed in the code. That and... I think its good now!

### CODE REVIEW

File: prototypes/files/f186_orwu1Newloc.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+                if (_.isEmpty(sliced) && rpcArgs[0] !== 0) {
+                    sliced = clinicList.slice(0, -1);
+                }
+                res = sliced.map(obj => `${obj.clinicIen}^${obj.name}`);
+            } else res = descending.map(obj => `${obj.clinicIen}^${obj.name}`);
+            return res;
+        },
+
+        toReturnValue(res) {
+            // console.log('wtf', res);
```
Comment (mfuroyama) [inline prototypes/files/f186_orwu1Newloc.js]: Looks good @sidjtd! One really minor thing...don't forget to clear out your commented console logging statement here!  I mean, really...WTF!?  😆

Comment (kekuso) [inline prototypes/files/f186_orwu1Newloc.js]: Day sanity check hahaha

Comment (sidjtd) [inline prototypes/files/f186_orwu1Newloc.js]: LOL I read this last night and was thinking to myself "dat Cardinal rule about pull requesting b4 you leave..... this is what I get for totally ignoring it"

**Comment by sidjtd** [comment]
Gonna merge this in if thats cool with all!

**Comment by mfuroyama** [comment]
Do it!!

**Pull request merged**